### PR TITLE
adding zenodo json file to accompany releases

### PR DESCRIPTION
**Pull request by orianac**
@andersy005 see here for json file - let me know if you think there are any issues w the keywords i used - especially `grants` on L47.

### CODE REVIEW

File: .zenodo.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    }
+  ],
+  "upload_type": "software",
+  "keywords": ["downscaling", "CMIP6", "bias-correction", "climate"],
+  "grants": [{ "id": "Microsoft AI for Earth" }],
+  "description": "The codebase here was used by [CarbonPlan](https://carbonplan.org) to develop a collection of datasets featuring climate projections from the Coupled Model Intercomparison Project Phase 6, bias-corrected and downscaled to the 0.25 degree resolution. The datasets include daily maximum and minimum temperatures and daily precipitation spanning the entire globe over the time period 1950-2100. Notably, the dataset collection includes results from multiple downscaling methods, allowing users to better evaluate the effects of downscaling for their specific application of interest. More details on the project are available in CarbonPlan’s [explainer article](https://carbonplan.org/research/cmip6-downscaling-explainer). Also, you can interactively explore the data through a [web tool](https://carbonplan.org/research/cmip6-downscaling) and access the data via an [example Jupyter notebook](https://github.com/carbonplan/cmip6-downscaling/blob/main/notebooks/accessing_data_example.ipynb)."
```
Comment (andersy005) [inline .zenodo.json:48]: @orianac, i'm not very confident about whether the markdown gets rendered. i've seen some version of this JSON file with HTML in it though (e.g. https://github.com/tensorflow/tensorflow/blob/master/.zenodo.json) --> https://zenodo.org/record/6574269

Comment (andersy005) [reply .zenodo.json:48]: more specifically the `notes` section: https://github.com/tensorflow/tensorflow/blob/360fe7abad4d8f0f3e286f15f9b96257a8a2b90c/.zenodo.json?rgh-link-date=2022-10-04T21%3A46%3A41Z#L12

Comment (andersy005) [reply .zenodo.json:48]: Perhaps we should convert the markdown to HTML before merging??

File: .zenodo.json [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    },
+    {
+      "name": "Joe Hamman",
+      "affiliation": "CarbonPlan",
+      "orcid": "0000-0001-7479-8439"
+    }
+  ],
+  "upload_type": "software",
+  "keywords": ["downscaling", "CMIP6", "bias-correction", "climate"],
+  "grants": [{ "id": "Microsoft AI for Earth" }],
```
Comment (andersy005) [inline .zenodo.json:47]: the `grants` section  looks good

File: .zenodo.json (hunk 1 above)
Comment (orianac) [reply .zenodo.json:48]: sounds great- will do right now

Comment (andersy005) [inline .zenodo.json:48]:
```suggestion
  "description": "he codebase here was used by <a href="https://carbonplan.org">CarbonPlan</a> to develop a collection of datasets featuring climate projections from the Coupled Model Intercomparison Project Phase 6, bias-corrected and downscaled to the 0.25 degree resolution. The datasets include daily maximum and minimum temperatures and daily precipitation spanning the entire globe over the time period 1950-2100. Notably, the dataset collection includes results from multiple downscaling methods, allowing users to better evaluate the effects of downscaling for their specific application of interest. More details on the project are available in CarbonPlan’s <a href="https://carbonplan.org/research/cmip6-downscaling-explainer">explainer article</a>. Also, you can interactively explore the data through a <a href="https://carbonplan.org/research/cmip6-downscaling">web tool</a> and access the data via an <a href="https://github.com/carbonplan/cmip6-downscaling/blob/main/notebooks/accessing_data_example.ipynb">example Jupyter notebook</a>."
```

**Review by andersy005 — COMMENTED** [review]
@orianac, do we still want to include `communities` here?

File: .zenodo.json (hunk 1 above)
Comment (orianac) [reply .zenodo.json]: @andersy005 this look okay?

File: .zenodo.json [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+  "description": "The codebase here was used by <a href=\"https://carbonplan.org\">Carbonplan</a> to develop a collection of datasets featuring climate projections from the Coupled Model Intercomparison Project Phase 6, bias-corrected and downscaled to the 0.25 degree resolution. The datasets include daily maximum and minimum temperatures and daily precipitation spanning the entire globe over the time period 1950-2100. Notably, the dataset collection includes results from multiple downscaling methods, allowing users to better evaluate the effects of downscaling for their specific application of interest. More details on the project are available in CarbonPlan's <a href=\"https://carbonplan.org/research/cmip6-downscaling-explainer\">explainer article</a>. Also, you can interactively explore the data through a [web tool](https://carbonplan.org/research/cmip6-downscaling) and access the data via an <a href=\"https://github.com/carbonplan/cmip6-downscaling/blob/main/notebooks/accessing_data_example.ipynb\"example Jupyter notebook</a>."
```
Comment (andersy005) [inline .zenodo.json:49]: one more suggestion.. you missed `web tool` :) 

```suggestion
  "description": "The codebase here was used by <a href=\"https://carbonplan.org\">Carbonplan</a> to develop a collection of datasets featuring climate projections from the Coupled Model Intercomparison Project Phase 6, bias-corrected and downscaled to the 0.25 degree resolution. The datasets include daily maximum and minimum temperatures and daily precipitation spanning the entire globe over the time period 1950-2100. Notably, the dataset collection includes results from multiple downscaling methods, allowing users to better evaluate the effects of downscaling for their specific application of interest. More details on the project are available in CarbonPlan's <a href=\"https://carbonplan.org/research/cmip6-downscaling-explainer\">explainer article</a>. Also, you can interactively explore the data through a <a href=\"https://carbonplan.org/research/cmip6-downscaling\">web tool</a> and access the data via an <a href=\"https://github.com/carbonplan/cmip6-downscaling/blob/main/notebooks/accessing_data_example.ipynb\"example Jupyter notebook</a>."
```
```

**Comment by andersy005** [comment]
pre-commit.ci autofix

**Pull request merged**